anni-playback/src/sources/cached_http/cache.rs

```diff
@@ -58,12 +60,15 @@ impl CacheStore {
 
         create_dir_all(path.parent().unwrap())?; // parent of `path` exists
 
-        File::options()
-            .read(true)
-            .append(true)
+        let _ = File::options()
```

Comment:
What about removing the file directly?

Reply:
Then we still need to handle the error that file is not exist (if using fs::remove_file). So I think the two approach has the same complexity and current one is a bit more efficient (just modify the file size).